app/main/views/users.py

```diff
@@ -40,16 +40,35 @@ def auth_user():
 
     if user is None:
         return jsonify(authorization=False), 404
+
     elif encryption.authenticate_user(json_payload['password'], user) and user.active:
         user.logged_in_at = datetime.utcnow()
         user.failed_login_count = 0
         db.session.add(user)
         db.session.commit()
 
         return single_result_response(RESOURCE_NAME, user), 200
+
     else:
         user.failed_login_count += 1
         db.session.add(user)
+        db.session.flush()
```

Comment:
Could bulk add these I think but no biggie https://stackoverflow.com/a/31205155

Reply:
We don't seem to use bulk inserts very widely across the API at the moment, and I'm not overly concerned by the performance loss of two separate INSERTs, but it's a fair point to make.